sys/newlib_syscalls_default/syscalls.c

```diff
@@ -662,3 +647,20 @@ clock_t _times_r(struct _reent *ptr, struct tms *ptms)
 
     return (-1);
 }
+
+/**
+ * @brief Wrapper for newlib's assert implementation
+ */
+NORETURN
+void __wrap___assert_func (const char *file, int line, const char *func, const char *expr)
+{
+    (void)file;
+    (void)line;
+    (void)func;
+    (void)expr;
+#ifdef DEBUG_ASSERT_VERBOSE
+    _assert_failure(file, line);
+#else
+    _assert_panic();
```

Comment:
This breaks with `DEVELHELP=0` when e.g. building `examples/basic/default` for `same54-xpro`

Reply:
Thanks for spotting this. I added the fix prior to using this, so that bi-secting does not need to get more pain than necessary.